### DISCUSSION TITLE
Please could you add this to Photopea

### DISCUSSION THREAD

**Discussion by TRONPHOTON43** [Acly/krita-ai-diffusion#905] [General]
Hi devs of this amazing plug in. There is a website called Photopea which is a really cool photo editor and it has plug in support. I was wondering if you could please please somehow add this onto there so it has this kind of GPU generative AI? Please consider it!

**Comment by lovemakerz** [comment]
I might as well wear it on photoshop (it would be a nose-thumbing to adobe too!), I'd understand, but photopea... no ^^
